### PR TITLE
Update version for log4j vulnerability

### PR DESCRIPTION
**Pull request by FriederikeHanssen**
## PR checklist

Closes #XXX 

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [ ] Remove all TODO statements.
- [ ] Emit the `versions.yml` file.
- [ ] Follow the naming conventions.
- [ ] Follow the parameters requirements.
- [ ] Follow the input/output options guidelines.
- [ ] Add a resource `label`
- [ ] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
    - [ ] `PROFILE=docker pytest --tag <MODULE> --symlink --keep-workflow-wd`
    - [ ] `PROFILE=singularity pytest --tag <MODULE> --symlink --keep-workflow-wd`
    - [ ] `PROFILE=conda pytest --tag <MODULE> --symlink --keep-workflow-wd`

### CODE REVIEW

**Comment by FriederikeHanssen** [comment]
@GCJMackenzie do you have time to review? The mutect tests take forever 🙈

File: tests/modules/gatk4/mutect2/main.nf [hunk 1]
```diff
@@ -100,10 +102,10 @@ workflow test_gatk4_mutect2_mitochondria {
...
     run_pon = false
     run_mito = true
-    interval_label = 'chr22'
-    fasta = file(params.test_data['homo_sapiens']['genome']['genome_fasta'], checkIfExists: true)
-    fai = file(params.test_data['homo_sapiens']['genome']['genome_fasta_fai'], checkIfExists: true)
-    dict = file(params.test_data['homo_sapiens']['genome']['genome_dict'], checkIfExists: true)
+    interval_label = 'chr21'
+    fasta = file(params.test_data['homo_sapiens']['genome']['genome_21_fasta'], checkIfExists: true)
+    fai = file(params.test_data['homo_sapiens']['genome']['genome_21_fasta_fai'], checkIfExists: true)
+    dict = file(params.test_data['homo_sapiens']['genome']['genome_21_dict'], checkIfExists: true)
```
Comment (GCJMackenzie) [inline tests/modules/gatk4/mutect2/main.nf:108]: change lines 96 and 97 to: 
[ file(params.test_data['homo_sapiens']['illumina']['mitochon_standin_recalibrated_sorted_bam'], checkIfExists: true)], 
and 
[ file(params.test_data['homo_sapiens']['illumina']['mitochon_standin_recalibrated_sorted_bam_bai'], checkIfExists: true)],
(apologies, wont let me comment on unchanged code lines)
Keep interval_label as 'chr22' and use the original genome_fasta , fai and dict files. This should shorten the mitochondria test immensely, I suspect this test is why the mutect2 tests are taking forever, the mitochon_standin bam file is far shorter so I kept it in to use here instead

**Review by GCJMackenzie — CHANGES_REQUESTED** [review]

File: tests/modules/gatk4/mutect2/main.nf (hunk 1 above)
Comment (FriederikeHanssen) [reply tests/modules/gatk4/mutect2/main.nf:108]: ah great, thank you :)

**Review by GCJMackenzie — APPROVED** [review]
Unsure why the mutect2 tests are taking so long, other than the test files being larger now. But at least its less than what it was before. Everything else looks good to me though, thanks for this :)

**Pull request merged**